app/src/main/java/deakin/gopher/guardian/view/GP/GPAddFragment.java

```diff
@@ -159,7 +159,7 @@ public View onCreateView(
               builder.setPositiveButton(
                   "YES",
                   (dialog, whichButton) -> {
-                    dataListener.onDataFinihsed(true);
+                    dataListener.onDataFinished(true);
                     // Toast.makeText(MainActivity.this, "Yaay", Toast.LENGTH_SHORT).show();
```

Comment:
This isn't part of your changes, but can you remove this silly commented code?

Reply:
Yup removed